cmd/kogito/command/iozip/zip_files.go

```diff
@@ -104,6 +105,29 @@ func zipFilesInDir(dir string, resource string, binaryBuildType flag.BinaryBuild
 	return filesFound, nil
 }
 
+func zipDir(filePath string, pathToZip string, info os.FileInfo, myZip *zip.Writer) error {
+	relPath := strings.TrimPrefix(filePath, pathToZip)
+	if info.IsDir() {
```

Comment:
isn't better to use paths.Join? (or something similar, i don't remember the exactly the function name)

Reply:
for the record: the filepath.join doesn't work in this case see: https://play.golang.org/p/fokvkaJH3GN
Also, the `*zip.Writer.Create` function we use have this description:
```
// Create adds a file to the zip file using the provided name.
// It returns a Writer to which the file contents should be written.
// The file contents will be compressed using the Deflate method.
// The name must be a relative path: it must not start with a drive
// letter (e.g. C:) or leading slash, and only forward slashes are
// allowed. To create a directory instead of a file, add a trailing
// slash to the name.
```